internal/broker/services_test.go

```diff
@@ -139,3 +299,25 @@ func assertPlanContainsPropertyInUpdateSchema(t *testing.T, plan domain.ServiceP
 		t.Errorf("plan %s does not contain %s property in Update schema", plan.Name, property)
 	}
 }
+
+func assertPlanContainsPropertyValuesInCreateSchema(t *testing.T, plan domain.ServicePlan, property string, wantedPropertyValues []string) {
```

Comment:
Could we use some asserts from the library?
We just compare string with array of strings.